Source/DataSources/GeometryUpdater.js

```diff
@@ -317,7 +318,8 @@ define([
      */
     GeometryUpdater.prototype.isFilled = function(time) {
         var entity = this._entity;
-        return this._fillEnabled && entity.isAvailable(time) && this._showProperty.getValue(time) && this._fillProperty.getValue(time);
+        var visible = this._fillEnabled && entity.isAvailable(time) && this._showProperty.getValue(time) && this._fillProperty.getValue(time);
+        return Boolean(visible); // if show or fill are undefined, true && undefined === undefined
```

Comment:
Would it be more clear to use `defaultValue(visible, false)` thoughout?  Then you don't have to have the comment explaining what's going on.

Reply:
Sure, that works